plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -33,18 +33,27 @@
     private final MySQLContainer<?> container;
     private final Closeable cleanup;
 
+    public final int majorVersion;
+    public final int minorVersion;
+    public final int patchVersion;
```

Comment:
What about adding public static constants for the versions to test - `LATEST_IMAGE`, `DEFAULT_IMAGE`?
That should be sufficient?